proto/proto-backplane-grpc/src/main/proto/deephaven/proto/table.proto

```diff
@@ -265,6 +265,13 @@ service TableService {
    * primary use case for this is the Deephaven Web UI.
    */
   rpc ComputeColumnStatistics(ColumnStatisticsRequest) returns (ExportedTableCreationResponse) {}
+
+  /**
+   * Returns a new table representing a sliced subset of the original table. The start row is inclusive,
+   * and the end row is exclusive. If a negative value is given, then the row is counted from the end of
+   * the table.
+   */
+  rpc Slice(SliceRequest) returns (ExportedTableCreationResponse) {}
```

Comment:
Looking at javadoc on `io.deephaven.engine.table.Table#slice`, I think we need to be a bit more explicit in using "position" which implies "position space" as opposed to "RowSet key space".